includes/abstracts/llms.abstract.post.data.php

```diff
@@ -0,0 +1,217 @@
+<?php
+/**
+ * Defines base methods and properties for querying data about LifterLMS Custom Post Types
+ *
+ * @package LifterLMS/Abstracts
+ *
+ * @since   [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * LLMS_Post_Data abstract
+ *
+ * @since   [version]
+ * @version [version]
```

Comment:
Additionally, we've recently updated our documentations standards to require `@version` tags *only* on file header docblocks. I know that LifterLMS has `@version` tags on a tremendous number of functions and methods but after reviewing our standards and looking at what's required to generate docs at developer.lifterlms.com we've realized that this tag is superfluous and it's preferable to record an individual method changelog with multiple `@since` tags for each method.

Therefore `@version` tags on methods, classes, and functions for new code should be excluded and over time we'll remove these from existing code as we update those files and functions.

Additionally, we reviewed the WP core doc standards and decided to copy the order of tags as they do, this adds the changelog tags (`@since`) above the `@param` and `@return` tags, see the example here:  https://github.com/gocodebox/lifterlms/blob/master/docs/documentation-standards.md#functions-and-class-methods

I know this means that using existing code as an example is unacceptable. It's a large project to bring the existing codebase into alignment but it's relatively easy to start implementing this on new code moving forward. Thanks for understanding and not being too upset about me nitpicking here!

Reply:
see
https://github.com/eri-trabiccolo/lifterlms/commit/a6c56af52885bc7f565de9017490a5e3a6874980

I've only fixed the "new class" documentation as per the request above, but I would have no problem to do it for the other two involved classes (LLMS_Course_Data, LLMS_Quiz_Data). I didn't do it only because didn't know if you wanted it now or in the future.

Also feel free to request any additional change. :)